specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/Clusters.json

```diff
@@ -632,12 +632,6 @@
           "description": "Minimum CapacityReservation value in GB.",
           "type": "integer",
           "format": "int64"
-        },
-        "maxCapacity": {
-          "readOnly": true,
-          "description": "Maximum CapacityReservation value in GB.",
-          "type": "integer",
-          "format": "int64"
```

Comment:
Remember I got similar PR last week to remove this property.  it is done by you or someone else in your team. not sure why not do it together in one PR